### PR TITLE
Fix unit tests

### PR DESCRIPTION
**Pull request by aman-v-singh**
## Steps taken to fix unit-tests
- Replace inactive URLs with the `postman-echo.com` URL.
- Comparing only the content and ignoring the whitespaces.

### CODE REVIEW

**Comment by aman-v-singh** [comment]
@VShingala The tests are passing now.

File: codegens/swift/test/unit/fixtures/testcollection/collection.json [hunk 1]
```diff
@@ -721,11 +721,11 @@
 					"raw": "Duis posuere augue vel cursus pharetra. In luctus a ex nec pretium. Praesent neque quam, tincidunt nec leo eget, rutrum vehicula magna.\nMaecenas consequat elementum elit, id semper sem tristique et. Integer pulvinar enim quis consectetur interdum volutpat."
 				},
 				"url": {
-					"raw": "https://mockbin.org/request/:action",
+					"raw": "https://postman-echo.com/request/:action",
```
Comment (VShingala) [inline codegens/swift/test/unit/fixtures/testcollection/collection.json:724]: `https://postman-echo.com/request/:action` gives 404. Is this correct?

**Review by VShingala — COMMENTED** [review]
Added one comment rest looks good.

File: codegens/swift/test/unit/fixtures/testcollection/collection.json (hunk 1 above)
Comment (aman-v-singh) [reply codegens/swift/test/unit/fixtures/testcollection/collection.json:724]: @VShingala It is a path variable 
the complete URL is like this

```
"url": {
	"raw": "https://postman-echo.com/request/:action",
	"protocol": "https",
	"host": [
		"postman-echo",
		"com"
	],
	"path": [
		"request",
		":action"
	],
	"variable": [
		{
			"key": "action",
			"value": "post"
		}
	]
},
```

Comment (VShingala) [reply codegens/swift/test/unit/fixtures/testcollection/collection.json:724]: @aman-v-singh Even with replacing value of `:action` to `post`, the request to the corresponding URL returns 404.

Comment (aman-v-singh) [reply codegens/swift/test/unit/fixtures/testcollection/collection.json]: @VShingala Fixed it by changing it to `https://postman-echo.com/:action`

**Review by VShingala — APPROVED** [review]

**Pull request merged**